livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/stt.py

```diff
@@ -91,9 +93,12 @@ def update_options(
         *,
         model: WhisperModels | GroqAudioModels | None = None,
         language: str | None = None,
+        prompt: str | None = None,
     ) -> None:
         self._opts.model = model or self._opts.model
         self._opts.language = language or self._opts.language
+        if prompt is not None:
```

Comment:
I think we need to update it back to None if a user wants to clear the previous prompt too.